Gemfile

```diff
@@ -3,10 +3,11 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'sinatra', require: nil
+gem 'ruby_speech', github: 'adhearsion/ruby_speech'
```

Comment:
This can be removed if `ruby_speech` is bumped

Reply:
This appears to be a new dependency, is it really necessary for Ruby 3 support?

Reply:
It's a dependency in the gemspec already, but it looks like ruby_speech hasn't been bumped.

https://github.com/adhearsion/ruby_speech/commit/7c28b0302ee11ee55a8f0ea99f480a93ccc0fddb

This commit allows later active support versions for ruby_speech but it's in the master branch.

Reply:
RubySpeech 3.0.1 has been published